go.mod

```diff
@@ -84,8 +84,7 @@ require (
 	gopkg.in/inf.v0 v0.9.1
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
-	gopkg.in/yaml.v1 v1.0.0-20140924161607-9f9df34309c0 // indirect
-	gopkg.in/yaml.v2 v2.2.2
+	gopkg.in/yaml.v2 v2.2.5
```

Comment:
Hey @tmjd - is an update to `yaml.v2` required for the fix, or was this just a byproduct of updating the `go-yaml-wrapper` pin?

Reply:
The fix was to update yaml.v2 to at least v2.2.4. I figured it would be good to stop using v1.0.0 also and found that mergo was the only thing that needed updating to remove the v1.0.0 inclusion.
Is there a hesitation around these updates?